tests/projects/cuda-arch/base/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 3.14)
```

Comment:
Is this the right minimum version of cmake for cuda-based tests in blt?

@white238 -- in a follow-up PR, we might want to ensure that all the declared minimum versions throughout blt are consistent/accurate.

Reply:
Also, is this note at the beginning of the BLT docs still accurate?

> BLT officially supports CMake 3.8 and above. However we only print a warning if you are below this version. Some features in earlier versions may or may not work. Use at your own risk.

Reply:
This is actually a very hard question to answer. It is nearly impossible to know what is the minimum and really depends on the project and what they are using.

Also it is not just the CMake version minimum. Depending on what you put there CMake activates different policies that can change how CMake behaves.